src/pycolmap/geometry/bindings.cc

```diff
@@ -71,6 +71,11 @@ void BindGeometry(py::module& m) {
   py::implicitly_convertible<py::array, Eigen::Quaterniond>();
   MakeDataclass(PyRotation3d);
 
+  m.def("get_covariance_for_rigid3d_inverse",
```

Comment:
Maybe make this a class method of Rigid3d (like `Inverse`) and name it `covariance_of_inverse` or so?